lms/envs/common.py

```diff
@@ -2631,17 +2631,33 @@ def _make_locale_paths(settings):
 AUTH_PASSWORD_VALIDATORS = [
     {
         "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        "OPTIONS": {
```

Comment:
Let's please make sure we have documented these changes in `Ironwood core changes`